### PR TITLE
Transparent Translations

### PR DESCRIPTION
**Pull request by Kenzso**
Your branch is up to date with 'origin/transparentTrnsl'.

 Changes to be committed:
	modified:   resources/views/localizations/app.blade.php

### CODE REVIEW

**Review by kdmnk — CHANGES_REQUESTED** [review]
Please make the changes as discussed offline.

**Comment by Kenzso** [comment]
The changes have been made, as we discussed 'offline' :D 
Well, at least I hope this is what we agreed on

**Comment by kdmnk** [comment]
Yes, thanks! :D

File: resources/views/localizations/app.blade.php [hunk 1]
```diff
@@ -79,23 +79,30 @@
                     @foreach ($expressions as $key => $value)
                     {{-- Not shown if the expression have an ongoing, unapproved change --}}
                     @php
-                    $duplicate = App\Models\LocalizationContribution::where('language', App::getLocale())->where('key',  $fname.'.'.$key)->where('approved', false)->first();
+                    $translations = App\Models\LocalizationContribution::where('language', App::getLocale())->where('key',  $fname.'.'.$key);
```
Comment (luksan47) [inline resources/views/localizations/app.blade.php:82]: Possible optimization: move this (and other php parts) to controller and use eager loading.

**Review by luksan47 — APPROVED** [review]
LGTM. Thanks for working on this!
Once @kdmnk gives you the +1, we can merge.

**Comment by Kenzso** [comment]
is this how you do that? I'm sorry, but I am not entirely sure

**Review by kdmnk — APPROVED** [review]
Yes, thanks!

**Comment by kdmnk** [comment]
@luksan47

**Pull request merged**